nipyapi/nifi/api_client.py

```diff
@@ -65,6 +65,7 @@ def __init__(self, host=None, header_name=None, header_value=None, cookie=None):
         """
         self.rest_client = RESTClientObject()
         self.default_headers = {}
```

Comment:
I think you want to revert this?

Reply:
This is still the correct place to fix it until we need regenerate the client.